ci/packages/package.go

```diff
@@ -0,0 +1,254 @@
+		return err
+	}
+	if err := sh.RunV("bin/build"); err != nil {
+		return err
+	}
+	if err := packageDebian("build/myst/myst", "amd64"); err != nil {
+		return err
+	}
+	return env.IfRelease(storage.UploadArtifacts)
+}
+
+// PackageLinuxDebianArm builds and stores debian arm package
+func PackageLinuxDebianArm() error {
+	logconfig.Bootstrap()
+	defer log.Flush()
+	if err := goGet("github.com/debber/debber-v0.3/cmd/debber"); err != nil {
+		return err
+	}
+	if err := sh.RunV("bin/build_xgo", "linux/arm"); err != nil {
+		return err
+	}
+	if err := packageDebian("build/myst/myst_linux_arm", "armhf"); err != nil {
+		return err
+	}
+	return env.IfRelease(storage.UploadArtifacts)
+}
+
+// PackageLinuxRaspberryImage builds and stores raspberry image
+func PackageLinuxRaspberryImage() error {
+	logconfig.Bootstrap()
+	defer log.Flush()
+	if err := goGet("github.com/debber/debber-v0.3/cmd/debber"); err != nil {
+		return err
+	}
+	if err := sh.RunV("bin/build_xgo", "linux/arm"); err != nil {
+		return err
+	}
+	if err := packageDebian("build/myst/myst_linux_arm", "armhf"); err != nil {
+		return err
+	}
+	return env.IfRelease(func() error {
+		if err := sh.RunV("bin/package_raspberry"); err != nil {
+			return err
+		}
+		return storage.UploadArtifacts()
+	})
+}
+
+// PackageOsxAmd64 builds and stores OSX amd64 package
+func PackageOsxAmd64() error {
+	logconfig.Bootstrap()
+	defer log.Flush()
+	if err := packageStandalone("build/myst/myst_darwin_amd64", "darwin", "amd64"); err != nil {
+		return err
+	}
+	return env.IfRelease(storage.UploadArtifacts)
+}
+
+// PackageWindowsAmd64 builds and stores Windows amd64 package
+func PackageWindowsAmd64() error {
+	logconfig.Bootstrap()
+	defer log.Flush()
+	if err := packageStandalone("build/myst/myst_windows_amd64.exe", "windows", "amd64"); err != nil {
+		return err
+	}
+	return env.IfRelease(storage.UploadArtifacts)
+}
+
+// PackageIOS builds and stores iOS package
+func PackageIOS() error {
+	logconfig.Bootstrap()
+	defer log.Flush()
+	if err := sh.RunV("bin/package_ios", "amd64"); err != nil {
+		return err
+	}
+	return env.IfRelease(storage.UploadArtifacts)
+}
+
+// PackageAndroid builds and stores Android package
+func PackageAndroid() error {
+	logconfig.Bootstrap()
+	defer log.Flush()
+	if err := sh.RunV("bin/package_android", "amd64"); err != nil {
+		return err
+	}
+	return env.IfRelease(storage.UploadArtifacts)
+}
+
+// PackageDockerAlpine builds and stores docker alpine image
+func PackageDockerAlpine() error {
+	logconfig.Bootstrap()
+	defer log.Flush()
+	if err := sh.RunV("bin/package_docker"); err != nil {
+		return err
+	}
+	if err := saveDockerImage("myst:alpine", "build/docker-images/myst_alpine.tgz"); err != nil {
+		return err
+	}
+	return env.IfRelease(storage.UploadDockerImages)
+}
+
+// PackageDockerUbuntu builds and stores docker ubuntu image
+func PackageDockerUbuntu() error {
+	logconfig.Bootstrap()
+	defer log.Flush()
```

Comment:
The same with `log.Flush()` maybe there is a way to do it once when mage finish his job?